index.md

```diff
@@ -1,16 +1,16 @@
-Tracking the progress of [OMEMO](http://conversations.im/omemo/)
+Tracking the progress of [OMEMO](https://conversations.im/omemo/)
 integration in various XMPP clients.
 
-[![OMEMO Logo](./images/omemo_logo.png)](http://omemo.top)
+[![OMEMO Logo](./images/omemo_logo.png)](https://omemo.top/)
 
 # Client Progress
 
 | **Client**  | Tracking Issue | Bounty | Work in Progress | Testing | Done |
 |-------------|----------------|--------|------------------|---------|------|
-| Conversations | [☑ #1](https://github.com/siacs/Conversations) |  | ☑ | ☑ | ☑ |
-| Gajim | [☑ #8161](https://dev.gajim.org/gajim/gajim/issues/8161) |  | ☑ | ☑ | ☑ |
+| Conversations | | ☐ | ☑ | ☑ | ☑ |
```

Comment:
In all fairness – I think Conversations was the first client to implement OMEMO. I wanted to give it at least some credit. 

@iNPUTmice Shall we pick another issue or just leave it empty? 😸